darts/utils/likelihood_models.py

```diff
@@ -0,0 +1,59 @@
+"""
+Likelihood Models
+-----------------
+"""
+
+from abc import ABC, abstractmethod
+from typing import Sequence, Optional, Tuple
+import torch
+import torch.nn as nn
+
+from ..timeseries import TimeSeries
+
+
+class LikelihoodModel(ABC):
```

Comment:
This is perfect, excellent way to represent probabilistic likelihoods. Great job!